up42/storage.py

```diff
@@ -279,7 +279,12 @@ def get_orders(
         url = urljoin(base_url, "?" + urlencode(params, doseq=True, safe=""))
 
         orders_json = self._query_paginated_endpoints(url=url, limit=limit)
-        logger.info(f"Got {len(orders_json)} orders for workspace {self.workspace_id}.")
+        logger_message = (
```

Comment:
This is also a misleading log that we changed to remove the workspace_id reference in cases where the get_orders are set to the account level.